docs/source/markdown/podman-network-create.1.md

```diff
@@ -114,6 +116,10 @@ The `macvlan` and `ipvlan` driver support the following options:
   - Supported values for `macvlan` are `bridge`, `private`, `vepa`, `passthru`. Defaults to `bridge`.
   - Supported values for `ipvlan` are `l2`, `l3`, `l3s`. Defaults to `l2`.
 
+#### **--route**=*route*
+
+A static route in the format \<destination in CIDR notaion>,\<gateway>,\<route metric (optional)>. This route will be added to every container in this network. Only available with the netavark backend. Can be specified multiple times if more than one static route is desired.
```

Comment:
```suggestion
A static route in the format \<destination in CIDR notation>,\<gateway>,\<route metric (optional)>. This route will be added to every container in this network. Only available with the netavark backend. It can be specified multiple times if more than one static route is desired.
```

Reply:
and do you need to backslash the `<`, but not the `>` symbols?

Reply:
No those "\\" are a typo (not sure how that happened...). I'll fix them.

Reply:
It's not xml. Here's an example route:

`--route 10.0.10.0/24,10.0.0.1,100`

Not sure if my notation here is the best way to describe that.